samza-api/src/main/java/org/apache/samza/operators/data/IncomingSystemMessage.java

```diff
@@ -23,54 +23,47 @@
 
 
 /**
- * This class implements a {@link Message} that encapsulates an {@link IncomingMessageEnvelope} from the system
- *
+ * A {@link Message} implementation that provides additional information about its input {@link SystemStreamPartition}
+ * and its {@link Offset} within it.
```

Comment:
Just want to clarify that the offset is only comparable and unique within the SSP of this message. Do you think it's worth pointing out (maybe phrased better), or is that pretty obvious?